tests/runtime_neutral/test_quality_debt_overlay_gate.py

```
@@ -0,0 +1,45 @@
+REPO_ROOT = Path(__file__).resolve().parents[2]
+
+
+class QualityDebtOverlayGateTests(unittest.TestCase):
+    @unittest.skipIf(shutil.which("powershell") is None, "PowerShell executable not available in PATH")
+    def test_quality_debt_overlay_gate_restores_policy_bytes(self) -> None:
+        policy_path = REPO_ROOT / "config" / "quality-debt-overlay.json"
+        script_path = REPO_ROOT / "scripts" / "verify" / "vibe-quality-debt-overlay-gate.ps1"
+        original_bytes = policy_path.read_bytes()
+
+        try:
+            completed = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)